tests/integration/test_pipenv.py

```diff
@@ -92,14 +92,9 @@ def test_proper_names_unamanged_virtualenv(PipenvInstance):
 def test_directory_with_leading_dash(raw_venv, PipenvInstance):
     with temp_environ():
         with PipenvInstance(chdir=True, venv_in_project=False, name="-project-with-dash") as p:
-            if "PIPENV_VENV_IN_PROJECT" in os.environ:
```

Comment:
These tests aren't about this variable and shouldn't impact the test run since it use `with temp_environ()`.